Sources/IdxPath.swift

```diff
@@ -0,0 +1,16 @@
+/*********************************************
+ *
+ * This code is under the MIT License (MIT)
+ *
+ * Copyright (c) 2016 AliSoftware
+ *
+ *********************************************/
+
+import Foundation
+
+/// Temporary alias for Foundation.IndexPath type to aid support of both Swift 2 & 3.
+#if swift(>=3.0)
+  public typealias IdxPath = IndexPath
+#else
+  public typealias IdxPath = NSIndexPath
+#endif
```

Comment:
Why not simply use `IndexPath` everywhere? (so that when Swift 3 is adopted and we drop Swift 2 in the future, there won't be any change)

```swift
#if swift(<3.0)
  public typealias IndexPath = NSIndexPath
#endif
```